src/client.js

```diff
@@ -564,9 +682,10 @@ class RPCClient extends EventEmitter {
   }
 
   /**
-   * Request to join the game the user is playing
-   * @param {User} user The user whose game you want to request to join
-   * @returns {Promise}
+   * Request to join the game the user is playing.
+   * @param {Snowflake} user The user whose game you want to request to join
+   * @returns {Promise<never>}
+   * @deprecated This command was removed
```

Comment:
no it wasn't

Reply:
i tried to execute said function, and i got a 'command not found' error

Reply:
should i mark these as resolved?